robot/impl/robot_reconfigure_test.go

```diff
@@ -1425,9 +1426,12 @@ func TestRobotReconfigure(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, c, test.ShouldEqual, 0)
 
-		test.That(t, eA.Tick(context.Background(), false, uint64(time.Now().UnixNano())), test.ShouldBeNil)
-		test.That(t, eB.Tick(context.Background(), true, uint64(time.Now().UnixNano())), test.ShouldBeNil)
-		test.That(t, eA.Tick(context.Background(), true, uint64(time.Now().UnixNano())), test.ShouldBeNil)
+		test.That(t, fakeboard.Tick(
+			context.Background(), eA.(*fakeboard.DigitalInterruptWrapper), false, uint64(time.Now().UnixNano())), test.ShouldBeNil)
+		test.That(t, fakeboard.Tick(
+			context.Background(), eB.(*fakeboard.DigitalInterruptWrapper), true, uint64(time.Now().UnixNano())), test.ShouldBeNil)
+		test.That(t, fakeboard.Tick(
+			context.Background(), eA.(*fakeboard.DigitalInterruptWrapper), true, uint64(time.Now().UnixNano())), test.ShouldBeNil)
```

Comment:
I think this makes these tests way too wrapped in the fakeboard implementation.

How about this - use the `Value` API here. Since it is a fakeboard we can have it return what `Tick` used to do, cc @cheukt 

Also @cheukt I don't think the encoder code changing is the only thing that made this test flaky - likely a lot of our board interface project has made them flakier. We'll continue adding bandaids but I worry we'll just make these tests weird and not that useful in the spirit of what they were going to test initially.

Reply:
Not sure if I understand what you mean by using `Value` instead. Currently the fake board value function just calls the `basicDigitalInterrupt` one which will only increment the value when `Tick `is called. Are you suggesting changing the fake board `Value` function to increment the count? Can do that but I worry that might affect other tests.

Reply:
I see.
@cheukt can we change this test not to include an encoder? It really does not have to have one.

Reply:
At the very least we can change it to use fake encoder and return whatever it needs from Position there.

Reply:
Ya we don't care, so we can just remove this test?

Reply:
just remove the bits pertaining to the ticks and digital interrupts (not the test itself), I can come in later and clean it up some more.

Reply:
Great, @oliviamiller please yeet them from this test.